doc/functions.md

```diff
@@ -10,4 +11,37 @@ checks for the ability to interact with a node via:
 
 supports
 * minetest game default if present
-* else polyfill
\ No newline at end of file
+* else polyfill
+
+
+## `register_stairs_slabs({def})`
+Creates stairs, slabs and sometimes more.
+
+Use with:
+xcompat.functions.register_stairs_slabs({
+            baseitem = "", --the item the nodes are based on
+            recipeitem = "", --the item the nodes are crafted with, will use baseitem if unset
+
+            name = "Willow",
+            slab_name = "", --slab name, if nil will use name
```

Comment:
I prefer the comments to be above the entry. Having them trailing the line like they are here in this section, seems a bit messy.

Reply:
and to be pedantic:
```lua
--<some disabled code>
-- Some comment
```
First line has no space after "--" generally used to disable code.
Second line has a space after "--" making it imediately visible as a comment and not some "TODO"/disabled code

Generally first variant has no indentation before "--" where as second one may have it.

This mod doesn't seem to follow this style (yet) so mark me pedantic and ignore as you wish :D